lib/lotus/cli.rb

```diff
@@ -75,6 +75,7 @@ def console
     method_option :database, aliases: ['-d', '--db'], desc: "application database (#{Lotus::Generators::DatabaseConfig::SUPPORTED_ENGINES.keys.join('/')})", default: Lotus::Generators::DatabaseConfig::DEFAULT_ENGINE
     method_option :architecture, aliases: ['-a', '--arch'], desc: 'application architecture (container/app)', default: Lotus::Commands::New::Abstract::DEFAULT_ARCHITECTURE
     method_option :application_name, desc: 'application name, only for container', default: Lotus::Commands::New::Container::DEFAULT_APPLICATION_NAME
+    method_option :template, desc: 'template engine', default: 'erb'
```

Comment:
So, let's be consistent and replace `template` to `template_engine` in `.lotusrc` too